underdocs-renderer/src/main/kotlin/underdocs/renderer/output/html/render/page/PageRenderer.kt

```diff
@@ -159,6 +170,21 @@ class PageRenderer(
                                           localStorage.setItem(UnderdocsTheme.KEY, themeChoice);
                                         })
                                     })();
+                                    
+                                    (function setSnackbar() {
+                                      const snackbar = document.querySelector('#theme-snackbar');
+                                      const isPolicyAccepted = localStorage.getItem(UnderdocsTheme.POLICY) === null ? false : true;
+                            
+                                      if (!isPolicyAccepted) {
```

Comment:
First: the ternary operator is a bit superfluous here, you can achieve the same by using

~~~~JavaScript
const isPolicyAccepted = localStorage.getItem(UnderdocsTheme.POLICY) !== null
~~~~

However, let's examine this in greater context! We have a negation inside the expression on the rhs of the assignment (`!== null`) and then a negation again as part of the condition (`!isPolicyAccepted`). Essentially, we are doing an unnecessary double negation, which puts a little bit of a noise into the code.

What do you think about something like this?

~~~~JavaScript
const shouldShowPolicySnackbar = localStorage.getItem(UnderdocsTheme.POLICY) === null

if (shouldShowPolicySnackbar) {
  // Do the thing.
}
~~~~

I find it more readable because of the lack of negations.

Reply:
You are right, your solution is so much better. Using the ternary operator was just me overthinking stuff (trying to be tooo explicit I guess). Great idea tho!

Reply:
No probs, it's quite easy to overlook such small details. Once the plugin system is in place, we will extract this JS code into a proper JS file, so we can lint it. Then, the linter will catch such stuff for sure.